package.json

```diff
@@ -1,35 +1,42 @@
 {
   "name": "loki-grpc-client",
   "version": "0.1.0",
-  "description": "Skeleton template for a balena TypeScript project",
-  "homepage": "https://github.com/balena-io/balena-typescript-skeleton#readme",
+  "description": "Loki GRPC client for Node.js",
+  "homepage": "https://github.com/balena-io-modules/node-loki-grpc-client#readme",
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "keywords": [
     "balena",
-    "typescript"
+    "typescript",
+    "loki",
+    "grpc",
+    "logging",
+    "logs"
   ],
   "author": "",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/balena-io/balena-typescript-skeleton.git"
+    "url": "git+https://github.com/balena-io-modules/node-loki-grpc-client.git"
   },
   "bugs": {
-    "url": "https://github.com/balena-io/balena-typescript-skeleton/issues"
+    "url": "https://github.com/balena-io-modules/node-loki-grpc-client/issues"
   },
   "files": [
-    "build/"
+    "build/",
+    "proto/"
   ],
   "scripts": {
+    "protoc:compile": "mkdir build && npx grpc_tools_node_protoc --js_out=import_style=commonjs,binary:./build --grpc_out=./build --plugin=protoc-gen-grpc=`which grpc_tools_node_protoc_plugin` -I ./proto ./proto/*.proto",
+    "protoc:types": "npx grpc_tools_node_protoc --plugin=protoc-gen-ts=./node_modules/.bin/protoc-gen-ts --ts_out=./build -I ./proto ./proto/*.proto",
     "clean": "rimraf build",
-    "build": "npm run clean && tsc",
+    "build": "npm run clean && npm run protoc:compile && npm run protoc:types && tsc",
     "lint": "balena-lint --typescript src tests",
     "lint-fix": "balena-lint --typescript --fix src tests",
     "test:node": "mocha -r ts-node/register --reporter spec tests/**/*.spec.ts",
     "test": "npm run build && npm run lint && npm run test:node",
     "test:fast": "npm run build && npm run test:node",
-    "prepack": "npm run build"
+    "prepack": "npm install && npm run build"
```

Comment:
Is there a reason we need `npm install` in the prepack?
BalenaCI will do that for us before publishing.

Reply:
This was a work around so that I could `npm install git+https://github.com/balena-io-modules/node-loki-grpc-client.git#loki-client` (and it would build correctly) while developing in `open-balena-api` until the package is published. Do you know an alternative to make a project git installable?

Reply:
I see.
Alternatively you could wait for balenaCI to publish to npm and install one of the pre-releases that get published. I can see that atm `npm info loki-grpc-client versions` will give you a few `0.1.0-loki-client-<commit>` versions to use.

The above mostly works if you are not rapidly prototyping.
In general I personally prefer copying the build directories over to open-balena-api's node_modules, either with a plain `cp` or even with `rsync`. Kind of medieval, but it feels faster that anything else I've tried so far.

Reply:
Thanks Thodoris that's exactly what I was looking for :+1: